res/lang/programme/programme_en.txt

```diff
@@ -114,6 +114,8 @@ ONLY_1_BROADCAST_POSSIBLE               = Only 1 broadcast possible!
 ONLY_X_BROADCASTS_POSSIBLE              = Only %X% broadcast possible!
 NO_MORE_BROADCASTS_ALLOWED              = No more broadcasts allowed!
 BROADCAST_ONLY_ALLOWED_FROM_X_TO_Y      = Broadcast only allowed from %X%:00 to %Y%:00!
+LIVE_PRODUCTION_FINISH_WILL_COST_X      = Live production finish will cost %X%
```

Comment:
Finishing the live production will cost?

Reply:
Es geht um den "Abschluss", nicht das "Abschliessen" - so "the finish", nicht "finishing". Oder?

Reply:
Es klingt für mich nur komisch. "Live production completion will cost"? Wenn Dir das aber auch nicht gefällt, kann es auch so bleiben.